src/FSLibrary/MaxTPSTest.fs

```diff
@@ -132,35 +132,110 @@ let maxTPSTest (context: MissionContext) (baseLoadGen: LoadGen) (setupCfg: LoadG
                 context.image
                 (if context.flatQuorum.IsSome then context.flatQuorum.Value else false)
 
+    // PayPregenerated requires node restart between failed iterations to ensure validity of the pregenerated transactions
```

Comment:
The tier 1 topology is under thirty nodes, right?

Reply:
yeah, other topologies we use vary in size quite a bit, so I don't want to break the test for those